src/appleseed/foundation/utility/stopwatch.h

```diff
@@ -155,7 +183,10 @@ inline Stopwatch<Timer>& Stopwatch<Timer>::measure()
 template <typename Timer>
 inline uint64 Stopwatch<Timer>::get_ticks() const
 {
-    return m_elapsed;
+    if (m_paused)
```

Comment:
I suggest writing a bunch of unit tests to make sure that pausing a stopwatch works in all cases. You can use a custom timer that returns predetermined values to make your test deterministic and robust.